solver/src/solver.rs

```diff
@@ -59,6 +53,46 @@ pub trait Solver: 'static {
     fn name(&self) -> &'static str;
 }
 
+/// A batch auction for a solver to produce a settlement for.
+#[derive(Clone, Debug)]
+pub struct Auction {
+    /// An ID that idetifies a batch within a `Driver` isntance.
+    ///
+    /// Note that this ID is not unique across multiple instances of drivers,
+    /// in particular it cannot be used to uniquely identify batches across
+    /// service restarts.
+    pub id: u64,
+
+    /// The collection of all liquidity that the solver can use.
+    pub liquidity: Vec<Liquidity>,
```

Comment:
Now that limit orders are separated from Liquidity, I was expecting `Auction` to have an `orders` field...

Reply:
It does. I created the PRs out of order, but this one comes first.